### DISCUSSION TITLE
opening pdfs when using emacs in wsl

### DISCUSSION THREAD

**Discussion by nasseralkmim** [emacs-citar/citar#133] [Q&A] [answered]
I'm having problems with windows file paths and the helm-bibtex function that opens the pdf.

I use zotero to manage my references and the .bib file has the pdf paths with windows standards.

When I run emacs inside wsl it seems that using the function ```(f-exists? path)``` on a windows path it does not find it.

I tried converting the path with the wsl command ```wslpath -u```, however run in some problems with ```\``` symbol.

Any quick solutions?

**Comment by bdarcus** [comment, marked as answer]
Are you saying you have no problems doing the same thing with `helm-bibtex`?

`Bibtex-actions` is mostly a wrapper around the same `bibtex-completion` backend, so I would think the issue would be there.

@denismaier (who is on windows, and I think using WSL ATM) - you know anything about this?

**Reply by nasseralkmim** [reply]
Yes, the problem can be solved within the ```helm-bibtex``` package. Should have posted there.

It is something related to windows paths been checked within the Linux system and not been recognized.

Thanks nonetheless.

**Reply by bdarcus** [reply]
I would just describe what happens with `helm-bibtex` in an issue at that repo. But the problem is probably with `bibtex-completion` and WSL.
